debian/changelog

```diff
@@ -1,3 +1,9 @@
+python-oq-engine (2.5.0-0~precise01) precise; urgency=low
+
+  [Daniele Viganò]
+  * Now the command `oq webui start` tries to open a browser tab
+    with the WebUI loaded
+
```

Comment:
The changelog message should go *above* the line `python-oq-engine (2.5.0-0~precise01) precise; urgency=low``